.github/workflows/build-full-application.yaml

```diff
@@ -0,0 +1,99 @@
+name: Build Full Application workflow
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: "The name of the environment to build into"
+        required: true
+        type: string
+      workspace:
+        description: "The name of the workspace to build the application into"
+        required: true
+        type: string
+      services:
+        description: "A list of the services to build"
+        required: false
+        default: ""
+        type: string
+      tag:
+        description: "The tag to build from"
+        required: false
+        default: ""
+        type: string
+      multi_domain:
+        description: "Domain to use for github runner role"
+        required: false
+        default: ""
+        type: string
+      workflow_timeout:
+        description: "Timeout duration in minutes"
+        required: false
+        default: 5
+        type: number
+      target_repo:
```

Comment:
More pedantry but I think this may be better named triggering_repo as it is the name of the repo tha that has triggered the workflow ... target I feel is less descriptive.
We should also change the description to be more ... descriptive - eg The name of the repository that has triggered the workflow"

Reply:
I think we settled on "target" as it is the "target" of the integration testing. Happy to change it though.

Reply:
Changed to : 'triggering_repo'